src/zcl_abapgit_repo.clas.abap

```diff
@@ -403,24 +413,24 @@ CLASS zcl_abapgit_repo IMPLEMENTATION.
   ENDMETHOD.
 
 
-  METHOD rebuild_local_checksums. "LOCAL (BASE)
+  METHOD rebuild_local_checksums.
 
-    DATA: lt_local     TYPE zif_abapgit_definitions=>ty_files_item_tt,
+    DATA:
+          lt_local     TYPE zif_abapgit_definitions=>ty_files_item_tt,
```

Comment:
no need to change this part, code is the same, formatting is a bit different, more diffs makes it more difficult to review

Reply:
In this particular case I'd keep it. I did it to compare `rebuild` in `base` and `online`. They were styled a bit differently which made the diff less clear. Now it is visible that the difference is just a couple of lines. I'm going to refactor it later (#2086). Let's keep this particular exception please (and 2 below)